gatelevel_readers/reconstruct_verilog/src/reconstruct_ram18kx2.h

```diff
@@ -340,12 +340,22 @@ struct TDP_RAM18KX2_instance {
     return sram;
   }
 
-  std::string get_init_i1(const std::string &init1, const std::string &parity1,
```

Comment:
The same fix has to happen in the other file:
https://github.com/os-fpga/Raptor_Tools/blob/74f2ab79214015eaa8b6150f066dfe0dd7485b92/gatelevel_readers/reconstruct_verilog/src/reconstruct_ram36k.h#L372

Reply:
Alain, work was not marked done, as I mentioned in the JIRA case the RAM36 has yet to be addressed, it's just that it was late/early in this morning and I wanted to speed testing with Integration tests (RAM18 tests would pass not RAM36)